packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -125,22 +132,24 @@ export class PincodeSet extends React.Component<Props, State> {
   render() {
     const { route } = this.props
     const isVerifying = route.params?.isVerifying
+    const changePin = route.params?.changePin
+
     const { pin1, pin2, errorText } = this.state
 
     return (
-      <SafeAreaView style={styles.container}>
+      <SafeAreaView style={changePin ? styles.changePinContainer : styles.container}>
         <DevSkipButton onSkip={this.navigateToNextScreen} />
         {isVerifying ? (
-          // Verify
           <Pincode
+            title={i18n.t('onboarding:pincodeSet.verify')}
             errorText={errorText}
             pin={pin2}
             onChangePin={this.onChangePin2}
             onCompletePin={this.onCompletePin2}
           />
         ) : (
-          // Create
           <Pincode
+            title={changePin ? 'Create a new PIN' : ' '}
```

Comment:
This `Create a new PIN` text should be on a file in the `locales` folder as well

Reply:
got it!